tests/component/test_running_command.py

```diff
@@ -0,0 +1,34 @@
+import os
+import secrets
+import subprocess
+from pathlib import Path
+
+import pytest
+
+PYTHON_ROOT_DIR = str(Path(__file__).parent.parent.parent.absolute())
+CURRENT_DIR = str(Path(__file__).parent.absolute())
+
+
+@pytest.mark.parametrize("run_path", [PYTHON_ROOT_DIR, CURRENT_DIR], ids=["python_rootdir", "home_dir"])
+def test_pipeline_runs_successfully(run_path):
```

Comment:
@snova-zoltanc this test should test the `python` command in both python-root, and in `tests/component` directory.